Specifications/Events.Store.MongoDB/Events/for_EventConverter/when_converting_to_event_log_event/an_external_event.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Machine.Specifications;
+
+namespace Dolittle.Runtime.Events.Store.MongoDB.Events.for_EventConverter.when_converting_to_event_log_event
+{
+    public class an_external_event
+    {
+        static CommittedExternalEvent committed_event;
+        static IEventConverter event_converter;
+        static MongoDB.Events.Event result;
+
+        Establish context = () =>
+        {
+            committed_event = committed_events.a_committed_external_event(0, 0);
+            event_converter = new EventConverter();
+        };
+
+        Because of = () => result = event_converter.ToEventLogEvent(committed_event);
+
+        It should_have_the_same_content = () => result.Content.ToString().ShouldEqual(committed_event.Content);
+        It should_represent_the_same_event = () => result.ShouldRepresentTheSameBaseEventAs(committed_event);
+        It should_not_be_applied_by_aggregate = () => result.Aggregate.WasAppliedByAggregate.ShouldBeFalse();
+        It should_not_come_from_event_horizon = () => result.EventHorizonMetadata.FromEventHorizon.ShouldBeTrue();
```

Comment:
Something is off here - "should_NOT_come_from_event_horizon" => "FromEventHorizon.ShouldBeTrue()"

Reply:
Copy-pasta... This should fix it.
```suggestion
        It should_come_from_event_horizon = () => result.EventHorizonMetadata.FromEventHorizon.ShouldBeTrue();
```